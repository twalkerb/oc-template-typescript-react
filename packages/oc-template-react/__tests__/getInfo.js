const getInfo = require("../lib/getInfo");

test("should return the correct info", () => {
  const info = getInfo();
  delete info.version;
  expect(info).toMatchSnapshot();
});
