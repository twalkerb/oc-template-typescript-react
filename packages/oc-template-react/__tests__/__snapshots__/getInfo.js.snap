// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return the correct info 1`] = `
Object {
  "externals": Array [
    Object {
      "global": "React",
      "url": "https://cdnjs.cloudflare.com/ajax/libs/react/15.6.1/react.min.js",
    },
    Object {
      "global": "ReactDOM",
      "url": "https://cdnjs.cloudflare.com/ajax/libs/react/15.6.1/react-dom.min.js",
    },
  ],
  "type": "oc-template-react",
}
`;
